gatsby-config.js

```diff
@@ -15,5 +20,17 @@ module.exports = {
                 useResolveUrlLoader: true,
             },
         },
+        {
+            resolve: "gatsby-plugin-root-import",
+            options: {
+                api: path.join(__dirname, "src/api"),
+                styles: path.join(__dirname, "src/styles"),
+                components: path.join(__dirname, "src/components"),
+                db: path.join(__dirname, "src/db"),
+                assets: path.join(__dirname, "src/assets"),
+                hooks: path.join(__dirname, "src/hooks"),
+                constans: path.join(__dirname, "src/constans"),
```

Comment:
Can we simplify?

Reply:
I was searching but I did't find anything